Gruntfile.js

```diff
@@ -236,7 +236,8 @@ module.exports = function(grunt) {
           buildCSS: false,
           paths: {
             'css-builder': '../build/css-builder',
-            'normalize': '../build/normalize'
+            'normalize': '../build/normalize',
+            'jquery': 'empty:'
```

Comment:
This looks rather strange to me - but I'm not a Grunt expert. What does "empty:" do?